scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -3294,7 +3296,7 @@ static bool antistall_scan(void)
 	u64 jiffies_now;
 
 	if (!enable_antistall)
-		return true;
+		return false;
```

Comment:
Should this be a separate patch?

Reply:
ahh @hodgesds this change is noop right? i.e. that's just the return to the timer callback which does nothing IIUC.

Reply:
suspect the option to disable antistall ought to be removed btw

Reply:
https://access.redhat.com/solutions/3901121 but ok.

Reply:
> https://access.redhat.com/solutions/3901121 but ok.

https://git.kernel.org/pub/scm/linux/kernel/git/tj/sched_ext.git/commit/?h=for-6.17&id=cb444006a625c60e6d4dd3753863c3c74f96aac3